dbms/programs/odbc-bridge/ColumnInfoHandler.cpp

```diff
@@ -93,11 +93,11 @@ void ODBCColumnsInfoHandler::handleRequest(Poco::Net::HTTPServerRequest & reques
     if (params.has("schema"))
     {
         schema_name = params.get("schema");
-        LOG_TRACE(log, "Will fetch info for table '" << schema_name + "." + table_name << "'");
+        LOG(TRACE) << "Will fetch info for table '" << schema_name << "." << table_name << "'";
```

Comment:
This will calculate arguments eagerly. But we need lazy evaluation depending on current log level.

Example:

`LOG(TRACE) << sleep(100);`
- should not sleep if the log level is less than trace.

`LOG(TRACE, "Will fetch info for table '" << schema_name << "." << table_name << "'");` - this can support lazy evaluation.